python/ray/cloudpickle/__init__.py

```diff
@@ -1,11 +1,47 @@
 from __future__ import absolute_import
 
+import os
+from pickle import PicklingError
 
 from ray.cloudpickle.cloudpickle import *  # noqa
 from ray.cloudpickle.cloudpickle_fast import CloudPickler, dumps, dump  # noqa
 
+
 # Conform to the convention used by python serialization libraries, which
 # expose their Pickler subclass at top-level under the  "Pickler" name.
 Pickler = CloudPickler
 
-__version__ = '1.6.0'
+__version__ = "1.6.0"
+
+
+def _warn_msg(obj, method, exc):
+    return (
+        f"{method}({str(obj)}) failed."
+        "\nTo check which non-serializable variables are captured "
+        "in scope, re-run the ray script with 'RAY_PICKLE_VERBOSE_DEBUG=1'.")
```

Comment:
Can we also suggest users to use `inspect_serializability` here?

Reply:
I chose not to, because often times the object could be wrapped in some library-specific code / dynamically generated. This means that the user will not be able to actually call `inspect` on that object.